test/kb_ea_utils_server_test.py

```diff
@@ -473,7 +554,9 @@ def getPairedEndLib_SetInfo(self, read_libs_basename_list, refresh=False):
 
     ### TESTS 0.1-0.4: get Fastq Stats
     #
-    def test_get_fastq_ea_utils_stats(self):
+    @patch('biokbase.workspace.client.Workspace.get_objects',
+           side_effect=AssertionError('\n\n*****  Replace get_objects() with get_objects2()'))
+    def test_get_fastq_ea_utils_stats(self, mock):
```

Comment:
The existing impl code is using get_objects2(), but is not using the output of the call. It is not clear how the output was intended to be used. I have removed the get_objects2() calls and the related tests.